src/components/feedback/FeedbackWidget.vue

```diff
@@ -273,6 +287,17 @@
         this.feedbackInputState = false;
       },
 
+      wordCount() {
+        if (!this.feedback) {
+          return false;
+        }
+
+        if (this.feedback?.trim()?.match(/\w+/g)?.length < 5) {
```

Comment:
This does not work as intended:
```js
const req = { body: {} };
validateReq(req) // => true
```

Set a fallback for the chain: `(req.body?.feedback?.trim()?.match(/\w+/g)?.length || 0) < 5`

Same in the server middleware, and ideally we wouldn't be duplicating the validation code but have it in just one place.